chainerui/__init__.py

```diff
@@ -40,6 +40,14 @@
 )
 
 
+def setup_db_revision_config():
```

Comment:
Is this function name good?
* `setup`? But I have no idea.
* `db_revision_config`? I recommend `db_migration_config`.

Reply:
I changed `get_db_migration_config`. 

"`setup`" is may be correct. If it the original, `alembic.ini` is put at the migration directory and read it when migration. However, ChainerUI shares migration DB with normal DB so hide alembic configuration information to avoid to write SQL address twice, and no `alembic.ini` in the module. When using migration, the ChainerUI module requires to **setup** alembic configuration every time.